src/Core/Checkout/Cart/CartPersister.php

```diff
@@ -15,6 +15,50 @@
 
 class CartPersister implements CartPersisterInterface
 {
+    private const CART_SQL = <<<'SQL'
```

Comment:
please use an variable instead of an constant.

Reply:
@J-Rahe I can change it but this is something that should not change, so a constant makes more sense?

Reply:
@J-Rahe I have now changed it to a variable but I still think a constant makes way more sense.

Reply:
its about continuity, everywhere else we use variables.